tools/phenomenal/isatools/mw2isa.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="isatab_validator" name="isatab_validator" version="1.0">
+  <description>Produce a validation report on an ISA-Tab ZIP archive</description>
```

Comment:
Thanks @proccaserra for all the changes. Could you please let me know if description of this tool makes sense with what is available on the `<help>...</help>` part of this XML? Seems to me that command, inputs, description and help conflict in some ways. For instance, the "mw_study_id" input is not used in the command.